mllib/src/test/scala/org/apache/spark/ml/clustering/LDASuite.scala

```diff
@@ -253,6 +253,14 @@ class LDASuite extends SparkFunSuite with MLlibTestSparkContext with DefaultRead
     val lda = new LDA()
     testEstimatorAndModelReadWrite(lda, dataset, LDASuite.allParamSettings,
       LDASuite.allParamSettings, checkModelData)
+
+    def checkModelDataWithDataset(model: LDAModel, model2: LDAModel, dataset: Dataset[_]): Unit = {
```

Comment:
* We don't need this inline method.
* Leave a comment to explain why we are doing this.